antora-playbook.yml

```
@@ -94,8 +94,8 @@ asciidoc:
     prod-prev-ver: 7.18
     prod-short: Che
     prod-upstream: Eclipse{nbsp}Che
-    prod-url-secure: +https+://__<che-host>__
-    prod-url: +https+://__<che-host>__:__<che-port>__
+    prod-url-secure: +https+://che-host
+    prod-url: +https+://che-host:che-port
```

```suggestion
    prod-url: https://che-host:che-port
```
As discussed, this doesn't work in Antora and must be handled by pass-through macros in the text.